### PR TITLE
Variable names conflict with reserved words

### PR DESCRIPTION
**Pull request by TerryHowe**
These variable names conflict with reserved words and the rest of the code just wasn't being used.

### CODE REVIEW

File: pkg/curatedpackages/packagecontrollerclient_test.go [hunk 1]
```diff
@@ -197,9 +197,9 @@ func TestEnableCuratedPackagesSuccess(t *testing.T) {
 			values = append(values, "cronjob.suspend=true")
 		}
 		tt.chartInstaller.EXPECT().InstallChart(tt.ctx, tt.chart.Name, ociURI, tt.chart.Tag(), tt.kubeConfig, "", valueFilePath, values).Return(nil)
-		any := gomock.Any()
+		gomockAny := gomock.Any()
 		tt.kubectl.EXPECT().
-			GetObject(any, any, any, any, any, any).
+			GetObject(gomockAny, gomockAny, gomockAny, gomockAny, gomockAny, gomockAny).
```
Comment (chrisdoherty4) [inline pkg/curatedpackages/packagecontrollerclient_test.go:202]: We could just stick `gomock.Any()` in here. Not sure this is particularly advantageous.

File: pkg/curatedpackages/packagecontrollerclient_test.go [hunk 2]
```diff
@@ -197,9 +197,9 @@ func TestEnableCuratedPackagesSuccess(t *testing.T) {
 			values = append(values, "cronjob.suspend=true")
 		}
 		tt.chartInstaller.EXPECT().InstallChart(tt.ctx, tt.chart.Name, ociURI, tt.chart.Tag(), tt.kubeConfig, "", valueFilePath, values).Return(nil)
-		any := gomock.Any()
+		gomockAny := gomock.Any()
```
Comment (TerryHowe) [inline pkg/curatedpackages/packagecontrollerclient_test.go:200]:
```suggestion
```

File: pkg/curatedpackages/packagecontrollerclient_test.go [hunk 3]
```diff
@@ -197,9 +197,8 @@ func TestEnableCuratedPackagesSuccess(t *testing.T) {
 			values = append(values, "cronjob.suspend=true")
 		}
 		tt.chartInstaller.EXPECT().InstallChart(tt.ctx, tt.chart.Name, ociURI, tt.chart.Tag(), tt.kubeConfig, "", valueFilePath, values).Return(nil)
-		any := gomock.Any()
 		tt.kubectl.EXPECT().
-			GetObject(any, any, any, any, any, any).
+			GetObject(gomockAny, gomockAny, gomockAny, gomockAny, gomockAny, gomockAny).
```
Comment (TerryHowe) [inline pkg/curatedpackages/packagecontrollerclient_test.go:201]:
```suggestion
			GetObject(gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any()).
```

File: pkg/curatedpackages/packagecontrollerclient_test.go (hunk 1 above)
Comment (TerryHowe) [reply pkg/curatedpackages/packagecontrollerclient_test.go]: Done.

**Review by chrisdoherty4 — APPROVED** [review]

**Review by vincentni — APPROVED** [review]
/lgtm
/approve

**Pull request merged**